api/apps/geoprocessing/src/modules/planning-units/planning-units.geo.entity.ts

```diff
@@ -2,21 +2,47 @@
  * @todo move the planningUnitsGeom entity to the api
  */
 import { ApiProperty } from '@nestjs/swagger';
-import { Column, Entity, PrimaryColumn } from 'typeorm';
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+import { Geometry } from 'geojson';
+import { defaultSrid } from '@marxan/utils/geo';
+
+export enum ShapeType {
+  Square = 'square',
+  Hexagon = 'hexagon',
+  Irregular = 'irregular',
+}
```

Comment:
I think I should blame myself for going a slightly different route for the API service and apidb enum for this - I think we should unify the enum, but I should be taking care of this to amend my own mistake

Reply:
It may not be a "mistake". All layers may not even want to share the same enum all over the place.